config/user-unistd.m4

```diff
@@ -0,0 +1,18 @@
+dnl #
+dnl # Check for SEEK_DATA - only used for cp_files/seekflood test case.
+dnl #
+AC_DEFUN([ZFS_AC_CONFIG_USER_UNISTD], [
```

Comment:
I would name this to be more specific and to match the output vars, probably `ZFS_AC_CONFIG_USER_UNISTD_SEEK_DATA`.